### PR TITLE
Clean up jpp_configurations.xml

### PR DESCRIPTION
**Pull request by pdbain-ibm**
Remove unused specifications, relocate the flags they define.

Fixes https://github.com/eclipse/openj9/issues/4800

Signed-off-by: Peter Bain <peter_bain@ca.ibm.com>

### CODE REVIEW

**Comment by pdbain-ibm** [comment]
This will simplify maintenance of classpaths/source directory specifications.

File: jcl/jpp_configuration.xml [hunk 1]
```diff
@@ -108,123 +108,11 @@
...
-		<parameter name="msg:outputdir" value="java.base/share/classes/com/ibm/oti/util"/>
-		<parameter name="jxerules:outputdir" value="java/lang"/>
-	</configuration>
-
 	<configuration
 		  label="JAVA11"
 		  outputpath="JAVA11/src"
-		  flags="Java11"
-		  dependencies="JAVA10"
+		  flags="Sidecar19-SE,Sidecar19-SE-OpenJ9,Sidecar19-SE_RAWPLUSJ9,Java10,Java11"
```
Comment (pshipton) [inline jcl/jpp_configuration.xml]: This is missing Sidecar18-SE-OpenJ9, DAA, Open-Module-Support. It shouldn't contain Sidecar19-SE_RAWPLUSJ9

Comment (pdbain-ibm) [reply jcl/jpp_configuration.xml]: Thanks for the feedback.  `Sidecar19-SE_RAWPLUSJ9` is used in System.java, so I have removed references to it.  FYI @JasonFengJ9 please review.

Comment (JasonFengJ9) [reply jcl/jpp_configuration.xml]: Removal of `RAWPLUSJ9` simplifies the `pConfig` flag maintenance, I am with it.

**Comment by pdbain-ibm** [comment]
Updated per reviewers' comments.  Restored `SIDECAR18-TOOLS-OPENJ9` which is required by DDR.

**Comment by pshipton** [comment]
How do you know the Panama change is good? @tajila are you ok with this change?

**Comment by pshipton** [comment]
jenkins test sanity plinux jdk11

**Comment by pdbain-ibm** [comment]
I ran the preprocessor in my workspace on the PANAMA spec with the original and modified configuration file.  The Panama pConfig fails to compile in either case, though the number of errors is smaller with the new configuration file.

File: jcl/jpp_configuration.xml [hunk 2]
```diff
@@ -308,7 +196,7 @@
 		  label="PANAMA"
 		  outputpath="PANAMA/src"
 		  flags="Panama"
-		  dependencies="SIDECAR19-SE-B136"
+		  dependencies="JAVA11"
```
Comment (tajila) [inline jcl/jpp_configuration.xml]: JAVA12 would be more appropriate

Comment (pshipton) [reply jcl/jpp_configuration.xml]: I you make this change then this can be merged.

**Comment by pshipton** [comment]
What was the last change pushed? git shows that nothing changed.

File: jcl/jpp_configuration.xml (hunk 2 above)
Comment (pdbain-ibm) [reply jcl/jpp_configuration.xml]: Done.  Thanks, Tobi

**Comment by pshipton** [comment]
jenkins test sanity zlinux jdk11,jdk12

**Comment by pshipton** [comment]
I suspect this is causing failure in the jdk8 builds, like the following. Reverting while we work it out.
```
Creating images/lib/ext/dtfj.jar
13:24:15  'u' flag requires manifest, 'e' flag or input files to be specified!
13:24:15  Usage: jar {ctxui}[vfmn0PMe] [jar-file] [manifest-file] [entry-point] [-C dir] files ...
13:24:15  Options:
13:24:15      -c  create new archive
13:24:15      -t  list table of contents for archive
13:24:15      -x  extract named (or all) files from archive
13:24:15      -u  update existing archive
13:24:15      -v  generate verbose output on standard output
13:24:15      -f  specify archive file name
13:24:15      -m  include manifest information from specified manifest file
13:24:15      -n  perform Pack200 normalization after creating a new archive
13:24:15      -e  specify application entry point for stand-alone application 
13:24:15          bundled into an executable jar file
13:24:15      -0  store only; use no ZIP compression
13:24:15      -P  preserve leading '/' (absolute path) and ".." (parent directory) components from file names
13:24:15      -M  do not create a manifest file for the entries
13:24:15      -i  generate index information for the specified jar files
13:24:15      -C  change to the specified directory and include the following file
13:24:15  If any file is a directory then it is processed recursively.
13:24:15  The manifest file name, the archive file name and the entry point name are
13:24:15  specified in the same order as the 'm', 'f' and 'e' flags.
13:24:15  
13:24:15  Example 1: to archive two class files into an archive called classes.jar: 
13:24:15         jar cvf classes.jar Foo.class Bar.class 
13:24:15  Example 2: use an existing manifest file 'mymanifest' and archive all the
13:24:15             files in the foo/ directory into 'classes.jar': 
13:24:15         jar cvfm classes.jar mymanifest -C foo/ .
13:24:15  
13:24:15  make[2]: *** [/Users/jenkins/workspace/PullRequest-Sanity-JDK8-osx_x86-64_cmprssptrs-OpenJ9/build/macosx-x86_64-normal-server-release/images/lib/ext/dtfj.jar] Error 1
13:24:15  make[2]: *** Waiting for unfinished jobs....
13:24:19  make[1]: *** [images] Error 2
13:24:19  make: *** [images-only] Error 2
```

**Pull request merged**